forge/test/models/pytorch/text/albert/test_albert.py

```diff
@@ -41,18 +42,26 @@ def test_albert_masked_lm_pytorch(size, variant, test_device):
     varaint_name = model_ckpt.replace("-", "_")
     compiled_model = forge.compile(model, sample_inputs=inputs, module_name=f"pt_{varaint_name}_masked_lm")
 
+    co_out = compiled_model(*inputs)
+    fw_out = model(*inputs)
+
+    co_out = [co.to("cpu") for co in co_out]
+    fw_out = [fw_out] if isinstance(fw_out, torch.Tensor) else fw_out
+
+    assert all([compare_with_golden_pcc(golden=fo, calculated=co, pcc=0.99) for fo, co in zip(fw_out, co_out)])
```

Comment:
can you use `compare_with_golden` instead of `compare_with_golden_pcc` because the first one is more general and handles both vector and scalar values.